website/theme/BlogTagsPostsPage/index.tsx

```diff
@@ -56,14 +54,17 @@ function BlogTagsPostPage(props: Props): JSX.Element {
             </Link>
             <div className="margin-vert--xl">
               {items.map(({ content: BlogPostContent }) => (
-                <BlogPostItem
+                <BlogPostProvider
                   key={BlogPostContent.metadata.permalink}
                   frontMatter={BlogPostContent.frontMatter}
                   assets={BlogPostContent.assets}
                   metadata={BlogPostContent.metadata}
+                  content={BlogPostContent}
                   truncated>
-                  <BlogPostContent />
-                </BlogPostItem>
+                  <BlogPostItem truncated>
+                    <BlogPostContent />
+                  </BlogPostItem>
+                </BlogPostProvider>
```

Comment:
I think we somehow lost the "Read more" button here (you can see it on the prod `/blog/` page

Reply:
Fixed that, it was `s/metadata.truncated/metadata.hasTruncateMarker/`